Reference/google/protobuf/any.pb.swift

```diff
@@ -108,18 +108,14 @@ import Foundation
 ///         "@type": "type.googleapis.com/google.protobuf.Duration",
 ///         "value": "1.212s"
 ///       }
-struct Google_Protobuf_Any: ProtobufGeneratedMessage, ProtobufProto3Message {
+struct Google_Protobuf_Any: ProtobufGeneratedMessage, ProtobufProto3Message, ProtoNameProviding {
   public var swiftClassName: String {return "Google_Protobuf_Any"}
   public var protoMessageName: String {return "Any"}
   public var protoPackageName: String {return "google.protobuf"}
-  public var jsonFieldNames: [String: Int] {return [
-    "typeUrl": 1,
-    "value": 2,
-  ]}
-  public var protoFieldNames: [String: Int] {return [
-    "type_url": 1,
-    "value": 2,
-  ]}
+  public static let _protobuf_fieldNames: FieldNameMap = [
+    1: .unique(proto: "type_url", json: "typeUrl", swift: "typeURL"),
+    2: .same(proto: "value", swift: "value"),
```

Comment:
I'm curious what you think about #149 then. The reason I'm itching to drop the Swift names is because they serve no purpose other than debugging—they're only used when generating the `debugDescription` and the mirror, and aren't used as input at all.

All three strings are unlikely to be the same for `more_than_one_word` proto field names, but the Swift identifier is likely to be identical to the JSON one in many cases, so your observation about them not adding much overhead in most cases is probably true. That being said, I don't think we mangle the names *so much* when we generate properties that users would be terribly confused if we used the proto names instead of the Swift names in `debugDescription` and `Mirror`. I want to toss out everything that we don't have a strong need for to keep things slim.